tools/pythonpkg/tests/fast/api/test_read_csv.py

```diff
@@ -399,6 +399,7 @@ def test_filelike_none(self, duckdb_cursor):
             res = duckdb_cursor.read_csv(obj, header=True).fetchall()
 
     def test_internal_object_filesystem_cleanup(self, duckdb_cursor):
+        @pytest.mark.skip(reason="depends on garbage collector behaviour, and sporadically breaks in CI")
```

Comment:
Annotations go before the function definition

Reply:
Thanks! Was busy sending this out that I had not properly checked. Thanks.